### PR TITLE
test(JestMigration): convert RadialBarChartSpec.js -> RadialBarChart.spec.tsx

### PR DESCRIPTION
**Pull request by proke03**
## Description
Migrates test/specs/chart/RadialBarChartSpec.js ->[test-jest/chart/RadialBarChart.spec.tsx](https://github.com/recharts/recharts/compare/master...proke03:test(JestMigration)/convert-RadialBarChartSpec?expand=1#diff-0dc0e6c7f7f508033c8ce4bce798e28c7ce4f2e49e3e657a285cc2070b63af00)

## Related Issue
#3184

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: test/specs/chart/RadialBarChartSpec.js [hunk 1]
```diff
@@ -1,218 +0,0 @@
...
-      >
-        <RadialBar minAngle={15} background clockWise dataKey="uv" isAnimationActive={false} />
-        <Legend />
-      </RadialBarChart>,
-    );
-    expect(wrapper.find('.recharts-legend-wrapper').length).to.equal(1);
-    expect(wrapper.find('.recharts-legend-item').length).to.equal(7);
-  });
-
-  // it('Renders tooltip when add a Tooltip element', () => {
```
Comment (nikolasrieble) [inline test/specs/chart/RadialBarChartSpec.js:184]: Please do try to get this test working as well. The fact that it is commented out merely signifies a TODO.

**Review by nikolasrieble — CHANGES_REQUESTED** [review]
Please also try to get the Tooltip test running. Have a look at how this is done in other migrated tests.

File: test-jest/chart/RadialBarChart.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        barSize={10}
+        data={data}
+      >
+        <RadialBar background dataKey="uv" isAnimationActive={false} />
+        <Tooltip />
+      </RadialBarChart>,
+    );
+
+    expect(container.querySelectorAll('.recharts-tooltip-wrapper')).toHaveLength(1);
+    expect(container.querySelectorAll('.recharts-default-tooltip')).toHaveLength(1);
```
Comment (nikolasrieble) [inline test-jest/chart/RadialBarChart.spec.tsx:174]: Interesting, does this work without firing events? Good enough for now though, thank you!

**Review by nikolasrieble — APPROVED** [review]
Thanks!

File: test-jest/chart/RadialBarChart.spec.tsx (hunk 2 above)
Comment (proke03) [reply test-jest/chart/RadialBarChart.spec.tsx:174]: Thank you!

Comment (ckifer) [reply test-jest/chart/RadialBarChart.spec.tsx:174]: The wrapper is technically always there it just isn't shown/doesn't have content

**Review by ckifer — APPROVED** [review]
LGTM, thanks!

**Pull request merged**